test/sql/decimal_functions_sql_test.cpp

```diff
@@ -0,0 +1,81 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// distinct_sql_test.cpp
```

Comment:
Run the formatter script to update header:
```
$ ./script/formatting/formatter.py --add-header  test/sql/decimal_functions_sql_test.cpp
```

Reply:
Addressed in latest commit